bundles/org.openhab.binding.nobohub/src/main/java/org/openhab/binding/nobohub/internal/NoboHubBridgeHandler.java

```diff
@@ -413,6 +413,8 @@ public Collection<WeekProfile> getWeekProfiles() {
     }
 
     public void setStatusInfo(ThingStatus status, ThingStatusDetail statusDetail, @Nullable String description) {
+        logger.debug("Updating Nobø Hub Status to '{}' detail '{}' and description '{}'", status, statusDetail,
```

Comment:
This logging is not needed as `updateStatus()` will log the same.

Reply:
Think I found the correct logger name now :+1: